tests/golos.vesting_test_api.hpp

```diff
@@ -13,22 +13,25 @@ struct golos_vesting_api: base_contract_api {
 
     //// vesting actions
     action_result create_vesting(name creator, std::vector<name> issuers = {}) {
-        return create_vesting(creator, _symbol, golos::config::control_name, issuers);
+        return create_vesting(creator, _symbol, issuers, golos::config::control_name);
     }
     action_result create_vesting(name creator, symbol vesting_symbol, std::vector<name> issuers = {}, name notify_acc = N(notify.acc)) {
-        authority auth(1, {});
-        for(auto issuer : issuers)
-            auth.accounts.emplace_back(permission_level_weight{.permission = {issuer, N(eosio.code)}, .weight = 1});
-            
-        if(std::find(issuers.begin(), issuers.end(), creator) == issuers.end())
-            auth.accounts.emplace_back(permission_level_weight{.permission = {creator, N(eosio.code)}, .weight = 1});
-            
-        std::sort(auth.accounts.begin(), auth.accounts.end(),
-            [](const permission_level_weight& l, const permission_level_weight& r) {
-                return std::tie(l.permission.actor, l.permission.permission) < std::tie(r.permission.actor, r.permission.permission);
-            });
-        _tester->set_authority(creator, golos::config::invoice_name, auth, "owner");
-        _tester->link_authority(creator, _code, golos::config::invoice_name, N(retire));
+        if (issuers.size()) {
+            authority auth(1, {});
+            for (auto issuer : issuers) {
+                auth.accounts.emplace_back(permission_level_weight{.permission = {issuer, N(eosio.code)}, .weight = 1});
+            }
+            if (std::find(issuers.begin(), issuers.end(), creator) == issuers.end()) {
+                auth.accounts.emplace_back(permission_level_weight{.permission = {creator, N(eosio.code)}, .weight = 1});
```

Comment:
Now we have a guarantee that auth.accounts isn't empty. Should creator be added to issuers in this case?

Reply:
for me, it's better to have test_api simple and direct, without hidden actions to have more controllable tests. actually, it works for my tests without setting auths. but if we already have issuers processing here, creator can be added too